source/robot_model/include/robot_model/Model.hpp

```diff
@@ -34,18 +36,18 @@ struct InverseKinematicsParameters {
 
 /**
  * @brief parameters for the inverse velocity kinematics function
- * @param alpha for the time optimization
- * @param epsilon minimal time for the time optimization
+ * @param alpha gain associated to the time slack variable
  * @param proportional_gain gain to weight the cartesian coordinates in the gradient
  * @param linear_velocity_limit maximum linear velocity allowed in Cartesian space (m/s)
  * @param angular_velocity_limit maximum angular velocity allowed in Cartesian space (rad/s)
+ * @param period of the control loop (ns)
  */
 struct InverseVelocityParameters {
   double alpha = 0.1;
-  double epsilon = 1e-2;
   double proportional_gain = 1.0;
   double linear_velocity_limit = 2.0;
   double angular_velocity_limit = 2.0;
+  std::chrono::nanoseconds dt = 1ms;
```

Comment:
Implicit conversion between chrono types from ms to ns (I know this is handled fine, but the readability suffers). 
```suggestion
  std::chrono::nanoseconds dt = 1000ns;
```